README.md

```diff
@@ -202,6 +202,16 @@ def some_function(_),
   multiline pipelines into IEx might result in a syntax error, as IEx will
   evaluate the first line without realizing that the next line has a pipeline.
 
+* Avoid pipeline operator just once.
```

Comment:
How about a more complete sentence?

"Avoid using the pipeline operator just once."